cli/azd/pkg/tools/docker/docker.go

```diff
@@ -263,6 +263,10 @@ func (d *Cli) CheckInstalled(ctx context.Context) error {
 	if !supported {
 		return &tools.ErrSemver{ToolName: d.Name(), VersionInfo: d.versionInfo()}
 	}
+	// Check if docker is running
+	if _, err := tools.ExecuteCommand(ctx, d.commandRunner, "docker", "ps"); err != nil {
+		return err
+	}
```

Comment:
We should return a better error message here.  The one you get does not tell them to check if their docker daemon is running.

Reply:
@wbreza, I have updated by your comments, please re-review.